Codigo/Protocol_Writes.bas

```diff
@@ -3721,7 +3721,7 @@ Public Sub WriteDatosGrupo(ByVal UserIndex As Integer)
 
 124                     If i = 1 Then
 126                         Call Writer.WriteString8(UserList(UserList( _
-                                    .Grupo.Lider).Grupo.Miembros(i)).Name & "(Líder)")
+                                    .Grupo.Lider).Grupo.Miembros(i)).name & "(Líder)")
```

Comment:
revert to Name or rename field to name

Reply:
the field its already called name, thats why vb6 keep changing it